analyses/cell-type-wilms-tumor-06/Dockerfile

```diff
@@ -16,7 +16,10 @@ ENV OPENSCPCA_DOCKER=TRUE
 ENV RENV_CONFIG_CACHE_ENABLED=FALSE
 
 # Install renv
-RUN R --no-echo --no-restore --no-save -e "install.packages('renv')"
+RUN R -e "install.packages('renv')"
+
+# Install Rhtslib first to prevent package installation errors
+RUN R -e 'BiocManager::install("Rhtslib")'
```

Comment:
When running non-interactively, `Rscript` is usually a preferred to the main `R` command
```suggestion
RUN Rscript -e "install.packages('renv')"

# Install Rhtslib first to prevent package installation errors
RUN Rscript -e 'BiocManager::install("Rhtslib")'
```

Reply:
Thanks for catching this, Josh!

Reply:
Tbh I don't understand all of this but I trust you ;) Thank you!!